src/Presentation/ActiveStateChecker.php

```diff
@@ -28,11 +28,27 @@ public function isActive(Item $item)
             );
         }
 
+        if($route = $item->getRoute()) {
+            return $this->getPartialRoute(Request::route()->getName()) === $this->getPartialRoute($route);
```

Comment:
Could `request()->routeIs()` work as well? Would work for active state e.g.`users.*`

Reply:
I think the idea of this was that I was using this package with Laravel 5.2 or 5.3, the routeIs() was introduced in 5.4. You assume correctly, when you have a menu item like admin.users.index or users.index for example, if you navigate to add/edit/view subpages, then it was checking for users.index === users.edit, which is definitely not true, so this partial route is probably a simpler version of routeIs(), I haven't checked Laravel's implementation.